dastardcommander/trigger_config.py

```diff
@@ -294,6 +315,40 @@ def pushedClearGroupTrigger(self):
         dummy = True
         self.client.call("SourceControl.StopTriggerCoupling", dummy)
 
+    @pyqtSlot()
+    def pushedClearDisabled(self):
+        changed = self.triggerBlocker.clear()
+        assert len(self.triggerBlocker.blocked) == 0
+        if changed:
+            self.updateDisabledList()
+
+    @pyqtSlot()
+    def updateDisabledList(self):
+        ndisabled = len(self.triggerBlocker.blocked)
+        if ndisabled == 0:
+            msg = "All channels are enabled"
+        elif ndisabled == 1:
+            msg = "One channel is disabled: {}".format(self.triggerBlocker.blocked[0])
+        else:
+            msg = "{} channels are disabled: {}".format(
+                ndisabled, ",".join(map(str, self.triggerBlocker.blocked))
+            )
+        self.disabledTextEdit.setPlainText(msg)
+        self.channelChooserChanged()  # update that text box
+
+    @pyqtSlot(int)
+    def blockTriggering(self, channelIndex):
```

Comment:
I dont see this ever being called or connected to?

Reply:
See[ dc.py:144](https://github.com/usnistgov/dastardcommander/blob/f5f5890eb0d6d2aa7ba061bc6dbda2e0ba53405b/dastardcommander/dc.py#L144). That slot is called whenever you click a channel's button (in the Observe tab) to turn it off. (Only off, not on.)